practiceCourseGenerator/build.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```

Comment:
My habit is to use:

```sh
#!/bin/bash --
```

The `--` tells Bash that no further arguments are to follow.  This is a kind of security feature, to prevent the possibility of your program getting additional flags passed into it somehow.